utility_functions/main.cpp

```diff
@@ -2,4 +2,9 @@
 
 int main() {
     // make sure to run your functions!
+    Sum(7,10);
```

Comment:
This has a return that isn't accounted for. Collect the return and print the result as well as the sent numbers to Sum()